app/src/main/AndroidManifest.xml

```diff
@@ -24,7 +28,7 @@
         android:usesCleartextTraffic="true"
         android:theme="@style/AppThemeNotActionBar"
         android:name=".BrowserApplication"
-        tools:ignore="UnusedAttribute">
+        tools:ignore="UnusedAttribute,DataExtractionRules">
```

Comment:
This might bite us later. We should use the data extraction rules. For example: https://searchfox.org/mozilla-mobile/source/android-components/samples/browser/src/main/AndroidManifest.xml#32

Reply:
Added.